submissions/Roka20012/js-dom/interactive.js

```diff
@@ -92,14 +89,14 @@ function checkStory({ target }) {
     if (target.tagName == "A") {
         for (let i = 0; i < nav.children.length; i++) {
```

Comment:
It isn't a good idea use directly tagName in you JS. Imagine if you would need to add another `<a>` element to your html. and then another. all of them inerhit this logic. Add class please.